docs/guides/pagination.md

```diff
@@ -4,4 +4,124 @@ title: Pagination
 
 # Pagination
 
-Documentation coming soon
+Pagination is a common use case in APIs to efficiently return some results of a relationship instead of all of them, since that may be unperformant.
+
+GraphQL as a spec [recommends cursor based pagination](https://graphql.org/learn/pagination/) and refers to [Relay's Connection Spec](https://relay.dev/graphql/connections.htm) for specific implementation details.
+
+Here we show a minimal example of how you can leverage Strawberry's generic Types
+to build the types required to comply with the relay spec.
+
+```python
+
+import base64
+from typing import List, Generic, TypeVar, Optional
+
+import strawberry
+from strawberry.arguments import UNSET
+
+
+GenericType = TypeVar("T")  # type: ignore warning as 'T' is too succint to be meaningful for humans
+
+
+@strawberry.type
+class Connection(Generic[GenericType]):
+    """Represents a paginated relationship between two entities
+
+    This pattern is using when the relationship itself has attributes.
+    In the facebook domain for example, a friendship between two persons
+    would be a connection that might have a friendshipStartTime
+    """
+    page_info: 'PageInfo'
+    edges: list['Edge[GenericType]']
+
+
+@strawberry.type
+class PageInfo:
+    """Pagination context to navigate objects with cursor-based pagination
+
+    Instead of a page and per_page one has a cursor of the last object
+    and fetches item starting from that one
```

Comment:
Not sure what 
> Instead of a page and per_page one

means

Reply:
Rephrased, let me know if it's more clear